src/backend/shared/models.py

```diff
@@ -0,0 +1,11 @@
+import json
+
+
+class Transaction:
```

Comment:
As of right now, this is a duplicate of the Transaction object within Blockchain. I think the blockchain only needs to receive the to/from/amount, whilst the services need the transaction ID and signature. This needs to be looked into later but for the purposes of the PR I think we can leave it